assets/config/0.4.0-coins.json

```diff
@@ -1493,6 +1493,29 @@
     "active": false,
     "currently_enabled": false
   },
+  "SCA": {
+    "coin": "SCA",
+    "name": "Scalaris",
+    "type": "UTXO",
+    "coingecko_id": "scalaris",
+    "coinpaprika_id": "sca-scalaris",
```

Comment:
can't find this ids, if SCA is not listed on gecko/paprika, this should be set to `test-coin`